pkg/services/auth/jwt/auth.go

```diff
@@ -42,13 +47,46 @@ func (s *AuthService) Init() error {
 	if err := s.initClaimExpectations(); err != nil {
 		return err
 	}
+	if s.Cfg.JWTAuthKeyFile == "" && s.Cfg.JWTAuthJWKSetFile == "" && s.Cfg.JWTAuthJWKSetURL == "" {
+		if err := s.detectJWKSUri(); err != nil {
+			s.log.Debug("JWKS uri not detected due to", "err", err)
```

Comment:
It can be discussed if Grafana should fail if this fails since I guess no one would be able to authenticate? Should at least be logged as an error (`s.log.Error(...)`.